### DISCUSSION TITLE
[Feature Request] net maui dropdown autocomplete

### DISCUSSION THREAD

**Discussion by HimanshuBansal112** [CommunityToolkit/Maui#1032] [New Feature Discussions]
**Description**
I would like to have a feature to make a dropdown in net maui in which user enters text, and according to text entered in between, it suggests what could be options for next text from given list. There will be several, one or no items which are shown according to typed text. I would also like if I have option in which programmer choose whether it search a text from beginning in each string or from mid-anywhere in each string in list. For example, "te" would give result both "ate" and "tear" if searching from mid-anywhere but would give result "tear", not "ate" if search from beginning. My older post with same feature request is at https://github.com/dotnet/maui/issues/13545

**Public API Changes**
```<Autocompletetypebox ItemsSource="{Binding Items}"/>```

This allows user to choose from text below from what they have entered.

**Use-Case**
This can be helpful to choose from a large list.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
